### PR TITLE
FS-1372 Add Update Project Name Logic

### PR DESCRIPTION
**Pull request by gio-karunakaran**
Project name field is not grabbing the project name from Project Information, adding the logic to update this once the user has answered this question in the form.

### CODE REVIEW

File: .flaskenv [hunk 1]
```diff
@@ -2,4 +2,5 @@ FLASK_APP=app.py
 FLASK_ENV=development
 FLASK_RUN_PORT=5000
 FLASK_RUN_HOST=localhost
-DATABASE_URL=postgresql://postgres:postgres@127.0.0.1:7890/fsd_app_store_test
+DATABASE_URL=postgresql://postgres:postgres@127.0.0.1:5432/application_store
+USE_LOCAL_DATA=True
```
Comment (gidsg) [inline .flaskenv:6]: Can we revert these changes? seem unrelated.

File: db/models/aggregate_functions.py [hunk 2]
```diff
@@ -186,6 +186,11 @@ def update_application_and_related_form(
     application = ApplicationsMethods.get_application_by_id(application_id)
     application.last_edited = func.now()
     form_sql_row = FormsMethods.get_form(application_id, form_name)
+    if form_sql_row.name == "project-information":
+        fields_array = form_sql_row.json[0]["fields"]
+        for key in fields_array:
+            if key["key"] == "project-name":
+                application.project_name = key["answer"]
```
Comment (gidsg) [inline db/models/aggregate_functions.py:193]: Could we use `filter` here instead for `for` iteration?

File: .flaskenv (hunk 1 above)
Comment (gio-karunakaran) [reply .flaskenv]: sorry! reverted back, forgot to change it after testing locally 👍🏾

File: db/models/aggregate_functions.py (hunk 2 above)
Comment (gio-karunakaran) [reply db/models/aggregate_functions.py]: we're only using this for loop once here and nowhere else in the codebase, so would it be unnecessary to separate this out into a sperate function to use in a filter?

**Review by harryyo — APPROVED** [review]
Great

File: db/models/aggregate_functions.py [hunk 3]
```diff
@@ -186,6 +183,16 @@ def update_application_and_related_form(
     application = ApplicationsMethods.get_application_by_id(application_id)
     application.last_edited = func.now()
     form_sql_row = FormsMethods.get_form(application_id, form_name)
+
+    if form_name == "project-information":
+        fields_array = question_json[1]["fields"]
+        for key in fields_array:
+            if key["key"] == "KAgrBz":
```
Comment (ramsharma-prog) [inline db/models/aggregate_functions.py:190]: As I am making changes to the forms, the Key might change. What are your thoughts on 
if key["key"] == "KAgrBz" and if key[ "title"] =="Project name":

**Comment by ramsharma-prog** [comment]
Looks good! Just a thought on something 🙂

File: db/models/aggregate_functions.py (hunk 3 above)
Comment (gio-karunakaran) [reply db/models/aggregate_functions.py:190]: thats a good idea! I like the sound of (if key["key"] == "KAgrBz" or if key[ "title"] =="Project name":) 👍🏾

**Review by adamdavies1 — APPROVED** [review]
LGTM

**Review by ramsharma-prog — APPROVED** [review]
LGTM
